package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "autod",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
you should not change the version in PR, I'll release a new version after merge this PR. thanks